src/main/resources/org/jenkinsci/plugins/ghprb/extensions/comments/GhprbBuildLog/help-logExceprtLines.html

```diff
@@ -0,0 +1,6 @@
+<div>
```

Comment:
In fact I only have "Build Status Messages" and "Update commit status during build" in my "Application Setup".